packages/core/src/browser/style/tabs.css

```diff
@@ -83,7 +83,7 @@ body.theia-editor-highlightModifiedTabs
 #theia-main-content-panel .p-TabBar .p-TabBar-tab.p-mod-current.theia-mod-active {
     background: var(--theia-tab-activeBackground);
     color: var(--theia-tab-activeForeground);
-    border-top: 1px solid var(--theia-tab-activeBorderTop);
+    border-top: 1px solid var(--theia-selection-background);
```

Comment:
i don't think it is corret, `theia-mod-active` should use `tab-activeBorderTop`

Reply:
I only did this to make sure the color for all the active bars are consistent, as it was discussed in the issue that it should be applicable on all parts, i.e. main editor, sidebar, the terminal. 
That's the reason why I changed the color to `selection-background`

Reply:
We should use color variables in semantically meaningful way, not randomly to make it look nicer. Later values of color variables can be adjuster to look it properly, i.e. you have to use the same value for different color variables in some theme to make it look the same in all places which you listed, but you cannot break semantic of color variables.

Reply:
Thankyou for clarifying the issue, I have reverted it back to the original.